### ISSUE TITLE
Ep7 second-to-last example: syntax question

### ISSUE THREAD

**Issue by megane0-0** [swcarpentry/shell-novice#988]
>> wc -l $(find . -name '*.dat') | sort -n

Isn't it ".." instead of "."?
wc -l $(find .. -name '*.dat') | sort -n

**Comment by gcapes** [comment]
The exercise looks ok to me. `find .. -name '*.dat'` would start the search from the parent directory.

**Comment by megane0-0** [comment]
I thought that at that time our working directory was ~/Desktop/data-shell/writing/ and .dat files were located ~/Desktop/data-shell/creatures/ so we didn't get any retults when we ran with single dots and we had 2 sorted output when we ran with double dots. So I suggested the previous post. But as you said, this is the exercise to explain what the script would do, so we don't need to care about what the output is when we ran that script where we are working. Sorry for my misunderstanding and thank you for your comment.
